### PR TITLE
	Update sample design

### PR DESCRIPTION
**Pull request by TsvetinaIvanova**

### CODE REVIEW

**Comment by didiepetrova** [comment]
:+1:

**Comment by TsvetinaIvanova** [comment]
Added DisallowScroll=true

File: sample-geolocation/index.html [hunk 1]
```diff
@@ -1,33 +1,103 @@
...
+					<label class="u-text-bold">Timestamp: </label>
+				</div>
+				<div class="col u-text-left">
+					<span id="timestamp" class="u-color-accent"></span>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="footer">
+		<span>Cordova Plugins Samples</span>
```
Comment (Mike4o) [inline sample-geolocation/index.html]: This should be:
<span>Feature Examples<span>

File: sample-geolocation/index.html [hunk 2]
```diff
@@ -1,33 +1,103 @@
...
+			</div>
+			<div class="row">
+				<div class="col u-text-right">
+					<label class="u-text-bold">Timestamp: </label>
+				</div>
+				<div class="col u-text-left">
+					<span id="timestamp" class="u-color-accent"></span>
+				</div>
+			</div>
+		</div>
```
Comment (Mike4o) [inline sample-geolocation/index.html]: Does the results div render okay in landscape? To me it seems that there are too many rows and perhaps we can think of an approach to visualize the result in a less space consuming way. This is just a suggestion.

**Comment by Mike4o** [comment]
Squash the commits in a single one

**Comment by TsvetinaIvanova** [comment]
Squashed the commits.

**Comment by Mike4o** [comment]
:+1:

**Pull request merged**